### PR TITLE
Add 'vendor' target to Makefile and run it

### PR DESCRIPTION
**Pull request by adrianreber**

### CODE REVIEW

**Comment by adrianreber** [comment]
@kolyshkin Does adding 'make vendor' make sense or is it unnecessary?

**Comment by rst0git** [comment]
I think it makes sense. This is similar to `make vendor` in [podman](https://github.com/containers/podman/blob/599b7d746a6ccad6ebbd70abb72be30bead094a6/Makefile#L272).

File: Makefile [hunk 1]
```diff
@@ -54,4 +54,9 @@ rpc/rpc.pb.go: rpc/rpc.proto
 stats/stats.pb.go: stats/stats.proto
 	protoc --go_out=. $^
 
-.PHONY: build test phaul-test test-bin clean lint
+vendor:
+	GO111MODULE=on go mod tidy
+	GO111MODULE=on go mod vendor
+	GO111MODULE=on go mod verify
```
Comment (rst0git) [inline Makefile:60]: Perhaps we should use `$(GO)` instead of `go`?

Comment (adrianreber) [reply Makefile]: done

**Review by rst0git — APPROVED** [review]
LGTM

**Pull request merged**